uaclient/livepatch.py

```diff
@@ -131,6 +131,11 @@ def status() -> Optional[LivepatchStatusStatus]:
             [LIVEPATCH_CMD, "status", "--verbose", "--format", "json"]
         )
     except exceptions.ProcessExecutionError as e:
+        # only raise an error if there is a legitimate problem, not just lack
+        # of enablement
+        if re.match("Machine is not enabled.", e.stderr):
```

Comment:
Is `re.match` what you want? The '.' is "any character" and hasn't been escaped. Maybe `"Machine is not enabled" in e.stderr` better fits what you're actually trying to do here?

Reply:
I agree, we'll fix it before releasing.